lib/jsdom/events/EventTarget.js

```diff
@@ -0,0 +1,189 @@
+﻿"use strict";
+
+var EventException = require("./EventException");
+
+var propEvents = Symbol();
+
+module.exports = EventTarget;
+
+function EventTarget() {
+  if (!this.__jsdom) {
```

Comment:
Oh I see, I forgot about Window inheriting from EventTarget :-/. Needs a comment then, with a link to the io.js bug.